mongo/src/test/java/org/axonframework/extensions/mongo/eventsourcing/eventstore/MongoOptionsFactoryTest.java

```diff
@@ -54,13 +58,16 @@ void testCreateMongoOptions_customSet() {
         factory.setConnectionTimeout(11);
         factory.setMaxWaitTime(3);
         factory.setSocketTimeOut(23);
-        factory.setThreadsAllowedToBlockForConnectionMultiplier(31);
+//        factory.setThreadsAllowedToBlockForConnectionMultiplier(31);
+
+        MongoClientSettings options = factory.createMongoOptions();
+        assertEquals(3, options.getConnectionPoolSettings().getMaxWaitTime(TimeUnit.MILLISECONDS));
+        assertEquals(23, options.getSocketSettings().getConnectTimeout(TimeUnit.MILLISECONDS));
+        assertEquals(9, options.getConnectionPoolSettings().getMaxSize());
 
-        MongoClientOptions options = factory.createMongoOptions();
-        assertEquals(3, options.getMaxWaitTime());
-        assertEquals(23, options.getSocketTimeout());
-        assertEquals(9, options.getConnectionsPerHost());
-        assertEquals(11, options.getConnectTimeout());
-        assertEquals(31, options.getThreadsAllowedToBlockForConnectionMultiplier());
+        // TODO connect time and socket timeout is identical
+//        assertEquals(11, options.getConnectTimeout());
+        // TODO deprecated
+//        assertEquals(31, options.getThreadsAllowedToBlockForConnectionMultiplier());
```

Comment:
We can drop this assertion as the `getThreadsAllowedToBlockForConnectionMultiplier` logic will be removed too.